lib/kwtool/crawler/keyword_results.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Kwtool.Crawler.KeywordResults do
+  import Ecto.Query, warn: false
+
+  alias Kwtool.Account.Schemas.User
+  alias Kwtool.Crawler.Queries.KeywordResultQuery
+  alias Kwtool.Repo
+
+  def search(_, params) when params == %{}, do: []
```

Comment:
You can match the params from function clause
```suggestion
  def search(_, %{}), do: []
```

Reply:
To match the empty maps in this case, I think we need `when params == %{}` https://stackoverflow.com/questions/33248816/pattern-match-function-against-empty-map